scripts/maintenance/image_triage.py

```diff
@@ -0,0 +1,106 @@
+import typer
+import asyncio
+import httpx
+from httpx import Response
+from typing import Dict, List
+import tqdm.asyncio
+import tqdm
+from collections import deque
+
+
+PREFIX_SERVICES_COMPUTATIONAL = "simcore/services/comp"
+PREFIX_SERVICES_DYNAMIC = "simcore/services/dynamic"
+
+
+async def _httpx_request(
+    registry: str, user: str, password: str, path: str
+) -> Response:
+    params = {}
+    if user and password:
+        params["auth"] = (user, password)
+
+    url = f"{registry}/v2/{path}"
+    async with httpx.AsyncClient() as client:
+        response = await client.get(url, **params)
+        response.raise_for_status()
+        return response
+
+
+async def _compile_registry_report(
+    registry: str, user: str, password: str
+) -> Dict[str, List[str]]:
```

Comment:
Since you are annotating everything, why not go an extra mile and define a ``TypedDict`` to describe a bit the content of this response instead of a cold ``Dict[str, List[str]]`` :-) ?

Reply:
I cannot use it in this case since the string key is not well defined but it changes. The data is something like:
```python
{
   "repo1": ["1.0.0","2.0.0"],
   "repo-2": ["1.2.3", "1.2.44"],
}